packages/paste-core/components/button/src/styles.ts

```diff
@@ -73,3 +74,22 @@ export const SizeStyles: {[key in ButtonSizes]: BoxStyleProps} = {
     borderWidth: 'borderWidth0',
   },
 };
+
+export const customButtonStyles = {
+  backgroundColor: 'colorBackgroundBusy',
+  borderRadius: 'borderRadius0',
+  borderWidth: 'borderWidth40',
+  borderColor: 'colorBorderError',
+  fontWeight: 'fontWeightLight',
+  variants: {
+    destructive: {backgroundColor: 'colorBackgroundDestructiveDarkest', color: 'colorTextWeaker'},
+    link: {padding: 'space40', borderRadius: 'borderRadiusCircle'},
+    destructive_link: {
+      padding: 'space40',
+      borderRadius: 'borderRadiusCircle',
+      backgroundColor: 'colorBackgroundDestructiveLight',
+      color: 'colorTextWarningStrong',
+      fontWeight: 'fontWeightBold',
+    },
+  },
+} as PasteCustomCSS;
```

Comment:
For test fixtures or mocks, it's better to not include them in source `src` files because they get compiled with the button package and delivered to customers via the compiled JS. 

Better to have a fixtures folder, or for something things small, just a constant in the test file itself if it's not shared. I also don't mind them being duplicate between stories and tests, they can even be different styles between the two assets

Reply:
Makes sense